src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/layout.html.twig

```diff
@@ -53,8 +53,8 @@
                             </li>
                         </ul>
                     {% endif %}
-                    {{ render(path('_profiler_search_bar')) }}
-                    {% include '@WebProfiler/Profiler/admin.html.twig' with { 'token': token } only %}
+                    {{ render(path('_profiler_search_bar', app.request.query.all)) }}
```

Comment:
@fabpot for search results `request` is not available, should the controller pass it to the view?